internal/webserver/handler-api.go

```diff
@@ -846,6 +856,41 @@ func (h *handler) apiUpdateAccount(w http.ResponseWriter, r *http.Request, ps ht
 	fmt.Fprint(w, 1)
 }
 
+// apiUpdateSettings is handler for PUT /api/accounts
+func (h *handler) apiUpdateSettings(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
```

Comment:
Are you up for moving this endpoint to the new API under `PATCH /api/v1/auth/account`? That endpoint will be for the current logged in user, so there's no custom validation required.